modules/local/samplesheet_check.nf

```diff
@@ -17,7 +17,7 @@ process SAMPLESHEET_CHECK {
     when:
     task.ext.when == null || task.ext.when
 
-    script: // This script is bundled with the pipeline, in phac-nml/iridanextexample/bin/
+    script: // This script is bundled with the pipeline, in phac-nml/clustersplitter/bin/
```

Comment:
I don't think this is used in the pipeline so could you remove this file?

Reply:
Removed in 3357071637ddac104e7159bdc0d794a725c1272a